Gemfile

```diff
@@ -11,3 +11,8 @@ if RUBY_VERSION.to_f < 2.3
   gem "nio4r",    "< 2.4"
   gem "nokogiri", "< 1.10.3"
 end
+
+if RUBY_VERSION.to_f > 2.4
+  gem "rubocop",           "~> 0.92"
+  gem "rubocop-packaging", "~> 0.5"
+end
```

Comment:
I like the fact that there's a specific Rubocop gem for packaging :) Less sold on the conditional for MRI being newer… but I guess there's not really a better way to manage this if I want to keep MRI 2.3 in the test matrix?

Reply:
Also: I'll have to give this rubocop-packaging library a shot in my other gems :)

Reply:
> I like the fact that there's a specific Rubocop gem for packaging :)

Yay! :)

> Less sold on the conditional for MRI being newer… but I guess there's not really a better way to manage this if I want to keep MRI 2.3 in the test matrix?

Unfortunately, that's right, there's no way to keep this for MRI 2.3 because the later versions of `rubocop-packaging` works with `rubocop` >= 0.89 and `ruby_version` >= 2.4.
Maybe you can still use `rubocop-packaging` v0.1.0 for 2.3.8, but it works best with 2.4 or later! 

That later versions have more cops and even support autocorrect for most of them! So it's better to keep it this way I think for now and remove this part of `if` clause as soon as the support for 2.3 is dropped!

> Also: I'll have to give this rubocop-packaging library a shot in my other gems :)

That'd be awesome! It will help us, downstream maintainers, esp. Debian and Ubuntu core devs, maintain the libraries very easily without any problems! ❤️ 
If you get stuck with something or feel the need to discuss anything, feel free to ping me there :)
The docs are available at https://docs.rubocop.org/rubocop-packaging/ for easy reference.